libs/tasks/data-access/src/lib/services/tasks-data.service.spec.ts

```diff
@@ -1,16 +1,56 @@
 import { TestBed } from '@angular/core/testing';
 
 import { TasksDataService } from './tasks-data.service';
+import { of } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
 
 describe('TasksDataService', () => {
   let service: TasksDataService;
+  let httpMock: HttpClient;
 
   beforeEach(() => {
-    TestBed.configureTestingModule({});
+    httpMock = {
+      get: () => {
+        return of({});
+      },
+    };
+  });
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [TasksDataService],
+      providers: [{ provide: http }],
+    });
     service = TestBed.inject(TasksDataService);
   });
 
   it('should be created', () => {
     expect(service).toBeTruthy();
   });
+
+  it('#getAll should return all records sorted by dueDate', () => {
```

Comment:
do not pass this test. You can use `it.skip`. Then you can remove comments from the bottom, but still use `// Todo` comment.